009_team_7_agent_design/009_team_7_agent_design.tex

```diff
@@ -1 +1,449 @@
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, With Treaties, T7Only, 2000days, 20food, High Openness, 434deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with high openness (434 deaths).}
+    \label{fig: High Openness}
+\end{figure}
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, With Treaties, T7Only, 2000days, 20food, Low Openness, 29deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with low openness (29 deaths).}
+    \label{fig: Low Openness}
+\end{figure}
+
+It can be observed that the agents collectively perform significantly better when openness is low, as a stabilising of the cumulative deaths curve can be seen for the agent with low openness. This can be explained by the fact that an agent with high openness will have a lower standard for accepting message requests and treaties - meaning they are more likely to agree to a request that is detrimental to their survival. Furthermore, an agent with a low openness value, will be less welcoming to a change in floor and will therefore behave more cautiously. This will also improve the agents chances of survival. 
+
+
+\subsubsection{Neuroticism}
+\label{subsubsec: Neuroticism}
+
+\Cref{fig: High Neuroticism} and \Cref{fig: Low Neuroticism} illustrate the results from simulations with agents with high neuroticism ($>$70) and low neuroticism ($<$30) respectively. All other traits are left as default.
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, With Treaties, T7Only, 2000days, 20food, High Neur, 154deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with high neuroticism (154 deaths).}
+    \label{fig: High Neuroticism}
+\end{figure}
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, With Treaties, T7Only, 2000days, 20food, Low Neur, 102deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with low neuroticism (102 deaths).}
+    \label{fig: Low Neuroticism}
+\end{figure}
+
+It can be seen that higher levels of neuroticism result in a greater number of agent deaths. This is expected as neuroticism results in volatility in an agents behaviour. 
+
+\newpage
+\subsubsection{Conscientiousness}
+\label{subsubsec: Conscientiousness}
+
+\Cref{fig: High Conscientiousness} and \cref{fig: Low Conscientiousness} illustrate the results from simulations with agents with high conscientiousness ($>$70) and low conscientiousness ($<$30) respectively. All other traits are left as default.
```

Comment:
There seems to be an error with the referencing here (I managed to stop it from happening by removing the spaces after 'fig:'). This may just be me being stupid though.